### ISSUE TITLE
Question: should StaticCrsGraph have a row() method?

### ISSUE THREAD

**Issue by aprokop** [kokkos/kokkos#1071]
I'm thinking that in some situations it would be convenient to have a `row()` method similarly to `KokkosSparse::CrsMatrix` except missing values. It should also bypass manual attempts like creating subviews. I would be willing to implement this if agreed upon.

**Comment by ibaned** [comment]
Book-keeping: this was implemented in #1072 which was merged. Waiting on a unit test too, but already technically in develop.

**Comment by aprokop** [comment]
Could somebody assign this to me so that I won't forget?

**Comment by ibaned** [comment]
done :)

**Comment by crtrott** [comment]
I opened a new issue for the UnitTest since this will be closed with the next promotion. Andrey can you comment in there with just a "I am here" or so so I can assign it to you?

**Comment by aprokop** [comment]
@crtrott Go ahead and assign it to me. And then close this one, as the new one supersedes it.

**Comment by ibaned** [comment]
@aprokop since you're not officially a Kokkos dev you have to post a comment on #1076 before he can assign it to you

**Comment by crtrott** [comment]
And this one will be closed with the next promotion to master. At each promotion all "InDevelop" issues will be closed.

**Comment by aprokop** [comment]
Is it OK to add a commit that pushes the feature to the current Trilinos until the next Kokkos snapshot?

**Comment by ibaned** [comment]
@aprokop the next snapshot is supposedly happening soon. Probably sometime next week?

**Comment by aprokop** [comment]
@ibaned Thanks, I can wait then.
